Showroom/Footer/index.jsx

```diff
@@ -1,37 +1,40 @@
 import React from 'react'
-import * as UI from '../../'
+import * as Button from '../../Button'
+import K from '../K'
+import Link from '../../Link'
+import * as Paragraph from '../../Paragraph'
+import * as Title from '../../Title'
+import {version} from '../../package.json'
```

Comment:
Can't you instead add a [DefinePlugin](https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin) which exposes `VERSION` and maybe even `GIT_HASH` ?